### PR TITLE
DOC Document standardisation of extension hooks

### PR DESCRIPTION
**Pull request by GuySartorelli**
## Issue
- https://github.com/silverstripe/silverstripe-framework/issues/11258

### CODE REVIEW

File: en/08_Changelogs/6.0.0.md [hunk 1]
```diff
@@ -259,7 +259,19 @@ This release includes a number of bug fixes to improve a broad range of areas. C
 
 ### Most extension hook methods are now protected {#hooks-protected}
 
-Core implementations of most extension hooks such as `updateCMSFields()` now have protected visibility. Formally they had public visibility which meant they could be called directly which was not how they were intended to be used. Extension hook implementations are still able to be declared public in project code, though it is recommended that all extension hook methods are declared protected in project code to follow best practice.
+Core implementations of most extension hooks such as `updateCMSFields()` now have protected visibility. Formerly they had public visibility which meant they could be called directly which was not how they were intended to be used. Extension hook implementations are still able to be declared public in project code, though it is recommended that all extension hook methods are declared protected in project code to follow best practice.
```
Comment (ChloeSartorelli) [inline en/08_Changelogs/6.0.0.md:262]: This is an unrelated typo fix - "formally" changed to "formerly"

**Review by emteknetnz — APPROVED** [review]
